<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Geoportal</class>
 <widget class="QMainWindow" name="Geoportal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Геопортал</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>440</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
       border-radius: 5px;
       background-color: rgb(67, 98, 92);
       font: 15pt &quot;Segoe UI Variable Text&quot;;
       padding: 10px;
}
QPushButton:hover {
    background-color: rgb(54, 77, 70); 
}</string>
    </property>
    <property name="text">
     <string>Запуск</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>v1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>v2</string>
     </property>
    </item>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menusettings">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="actionchoose_path"/>
   </widget>
   <addaction name="menusettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionchoose_path">
   <property name="text">
    <string>Задать путь... </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
